app/experimenter/nimbus-ui/src/components/PageEditOverview/index.tsx

```diff
@@ -27,6 +27,10 @@ const PageEditOverview: React.FunctionComponent<PageEditOverviewProps> = () => {
   const refetchReview = useRef<() => void>();
   const [isServerValid, setIsServerValid] = useState(true);
 
+  const optionalStringBool = (value: string): boolean | null => {
+    return value.length ? value === "true" : null;
+  };
```

Comment:
Might be worth budding this off into a small library in lib/ along with `optionalBoolString` just for symmetry.